org.eclipse.lsp4e/src/org/eclipse/lsp4e/LanguageServerWrapper.java

```diff
@@ -975,7 +975,7 @@ void unregisterCommands(List<String> cmds) {
 		}
 	}
 
-	int getVersion(URI uri) {
+	public int getVersion(URI uri) {
```

Comment:
Before making it a public API, I think we need clear documentation, and/or clear name, about which version we're mentioning here. Is it the one for LSP's TextDocument object, or the one for the underlying Eclipse IDocument? I wold suggest
```java
/**
 * return the TextDocument version, suitable to build a TextDocumentIndentifier
 */
public int getTextDocumentVersion(URI uri) {
  ...
}
```,
Or maybe returning a `VersionedTextDocumentIdentifier` here, to clarify what it's meant to be used for.